packages/gatsby-theme-bodiless/create-node.js

```diff
@@ -18,9 +18,12 @@
 const pathUtil = require('path');
 const slash = require('slash');
 const crypto = require('crypto');
-const fs = require('fs-extra');
+const fs = require('fs');
```

Comment:
This import could be removed as it's not used anymore.

Reply:
@lucas-varela actually the `fs` is required by `listRemotes` on line 81.

I renamed `fs-extra` to `fse` in order to introduce `fs` here.